activity.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
The `#!` is not required, remove it instead of changing.

Reply:
Ok will remove it.